cypress/tests/organization-users.spec.ts

```diff
@@ -0,0 +1,258 @@
+import cyTranslate from '../support/translations';
+
+import LoginActions from '../support/Actions/auth/LoginActions';
+import NetworkActions from '../support/Actions/common/NetworkActions';
+
+import { API_ROUTES } from '../support/api.routes';
+import { APPLICATION_ROUTES } from '../support/application.routes';
+
+// Grouped test Actions
+const loginActions = new LoginActions();
+const networkActions = new NetworkActions();
+
+// User 1 login credentials
+const username = Cypress.env('TEST_USERNAME');
+const password = Cypress.env('TEST_PASSWORD');
+
+// User 2 username
+const username2 = 'cypress2';
+const inviteEmail = 'cypress2+1@appknox.io';
+const teamName = 'mfva';
+
+const NETWORK_WAIT_OPTS = {
+  timeout: 60000,
+};
+
```

Comment:
add `const ELEMENT_WAIT_OPTS = { timeout: 10000 };` and use for both 5000 and 10000 timeouts